Docs 5509 env variables v4 1

Cristian-Venticinque opened the pull request.

Commit:
Fixed wording.

Commit:
Udpdated screenshots.

Commit:
Removed outdated screenshots.

Commit:
Used proper screenshots in article. Minor wording updates.

Commit:
Updated screenshot called from different article.

Commit:
Clarified usage and differences when deploying from Studio vs Standalone

elrodro83 commented on modules/ROOT/pages/setting-environment-variables.adoc:
```diff
@@ -4,21 +4,43 @@ include::_attributes.adoc[]
 
-. Under *Run*, select *Run Configurations...*.
-. Select the corresponding run configuration. If a run configuration does not exist for your project, you can create one by using the new launch configuration button:
+When you deploy applications on-premises, set the environment variables from the command line when you start the Mule instance.
```
variables may also be set in the `wrapper.conf` file

elrodro83 commented on modules/ROOT/pages/setting-environment-variables.adoc:
```diff
@@ -4,21 +4,43 @@ include::_attributes.adoc[]
+
+== Set Environment Variables for On-Premises Deployments
+
+Append `-M-Dmule.<variable-name>=<value>` to the `mule start` command to set an environment variable. +
```
variables may also be set in the `wrapper.conf` file

elrodro83 reviewed (changes requested):
On a general note, what here is referred to as `environment` properties i usually hear it referred to as `system`. What is the correct wording?

kevintroller reviewed (approved):
LGTM

Commit:
Improved details about system properties usage.

-Clarified that system propertiues can be defined in the wrapper.conf 
file or through the command line when starting Mule.

Commit:
Added explanation about how to reference variables. 

-Also linked to the system properties doc, which explains how to define 
variables for on-premises deployments.

Commit:
Updated Studio screenshot.

elrodro83 commented on modules/ROOT/pages/mule-app-properties-system.adoc:
```diff
@@ -15,21 +15,45 @@ You can manage system properties in these ways:
+=== Set System Properties in the wrapper.conf File
 
-`<logger message="${propertyFromJVMArg}" doc:name="System Property Set in Studio through JVM args"/>`
+Add `wrapper.java.additional.n` entries to the `wrapper.conf` file in the `${MULE_HOME}/conf` directory, prepending property names with `-D`.
```
`wrapper.java.additional.n` -> `wrapper.java.additional.<n>`

elrodro83 commented on modules/ROOT/pages/mule-app-properties-system.adoc:
```diff
@@ -15,21 +15,45 @@ You can manage system properties in these ways:
+For example, to set Mule's encoding:
 
-For the Mule standalone, you can set and change property placeholder values through a JDK system property from the command line, for example:
+Add `wrapper.java.additional.1=-Dmule.encoding=ISO-8859-1` to the Wrapper configuration file.
```
`1` is already taken for sure, put a high number (999, for instance)

elrodro83 commented on modules/ROOT/pages/mule-app-properties-system.adoc:
```diff
@@ -15,21 +15,45 @@ You can manage system properties in these ways:
 
-`mule -M-Dsmtp.username=JSmith -M-Dsmtp.password=ChangeMe`
+[WARNING]
+If you add `wrapper.java.additional.n` entries to the configuration file, always change each instance of `n` to a consecutive number, or Java does not parse the properties correctly.
```
`wrapper.java.additional.n` -> `wrapper.java.additional.<n>`

elrodro83 replied:
the number need not be consecutive, just not rpeeated

Commit:
Review: Fixed format and example.
